src/main/java/baseball/game/Game.java

```diff
@@ -0,0 +1,87 @@
+package baseball.game;
+
+import baseball.player.Computer;
+import baseball.player.User;
+import camp.nextstep.edu.missionutils.Console;
+
+public class Game {
+    private boolean playing;
+    private int strikeCnt;
+    private int ballCnt;
+
+    static int FINISH_THE_GAME = 2;
+    static int COUNT_OF_INPUT_NUMBERS = 3;
+
+    private void init(){
+        strikeCnt = 0;
+        ballCnt = 0;
+    }
+
+    public void setGame(){
+        playing = true;
+    }
+
+    public boolean isPlaying(){
+        return playing;
+    }
+
+    public void run(){
+        Computer computer = new Computer();
+        User user = new User();
+
+        setGame();
+        System.out.println(Print.START_GAME);
+
+        while (isPlaying()){
+            computer.setComputerObject();
+
+            while ( !(isThreeStrike()) ){
+                System.out.print(Print.GET_INPUT);
+                user.setUserObject();
+
+                gameLogic(computer, user);
+                System.out.println(getGameResult());
+            }
+            askReplay();
+        }
+    }
+
+    public void gameLogic(Computer computer, User user){
+        init();
+
+        for(int index = 0; index < COUNT_OF_INPUT_NUMBERS; index++){
+            if(computer.getComputerNumber(index) == user.getUserNumber(index)){
+                strikeCnt++;
+            } else if(computer.isComputerContains(user.getUserNumber(index))){
+                ballCnt++;
+            }
+        }
```

Comment:
```suggestion
        for(int index = 0; index < COUNT_OF_INPUT_NUMBERS; index++){
            int userNumber = user.getUserNumber(index);
            if(computer.getComputerNumber(index) == userNumber){
                strikeCnt++;
            } else if(computer.isComputerContains(userNumber)){
                ballCnt++;
            }
        }
```
비교 로직에서 위 처럼 변수를 따로 빼는 방법도 좋을 수 있습니다. 😆

Reply:
오!! 코드가 좀 더 깔끔 해 지네요! 상세한 리뷰 감사합니다. 😊

Reply:
else if 말고 그냥 이프문으로 두개 바로 리턴해도 좋을 거 같아요!
`
if(computer.getComputerNumber(index) == userNumber){
    strikeCnt++;
}
if(computer.isComputerContains(userNumber)){
    ballCnt++;
}`